app/translations.js

```diff
@@ -496,6 +497,7 @@ const translations = {
       '{number, plural, =0 {0 Minuten} one {1 Minute} other {{number} Minuten}}',
     'off-canvas-info':
       'Mit einer Anmeldung können Sie Favoriten speichern und auf anderen Geräten wieder verwenden.',
+    open: 'geöffnet',
```

Comment:
I don't see a message "Schließt bald" (Closing soon) which we should display in case...